src/app/services/planqk-platform-login.service.ts

```diff
@@ -25,6 +25,11 @@ export class PlanqkPlatformLoginService {
   }
 
   public logoutFromPlanqkPlatform(): void {
-    this.keycloak.logout();
+    /* This is a (bad) workaround for the issue with logging out from PlanQK -
```

Comment:
I removed the workaround since I just realized it doesn't work anyway. That means until the angular upgrade is finished, logging out requires a separate manual logout from planqk.